adapters/google_cloud_storage.go

```diff
@@ -136,6 +139,24 @@ func (gcs *GoogleCloudStorage) GetObject(key string) ([]byte, error) {
 	return ioutil.ReadAll(r)
 }
 
+// Function tries to create temporary file and remove it.
+// Returns OK if file creation was successfull.
```

Comment:
Let's write comments to functions in single format like:
```
//NameOfFunction tries to create ....
//returns nil if ok and error ....
```

Because this function doesn't return OK, it returns nil or error. Comments style is described [here](https://blog.golang.org/godoc).